src/__test__/App.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import App from '../App';
+
+describe('App page is working fine', () => {
+  test('renders App', () => {
+    const tree = render(
+      <Provider store={store}>
+        <App />
+      </Provider>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] This is currently failing. I will suggest you fix it.

![image](https://user-images.githubusercontent.com/51296741/152619173-81ba40be-713d-4038-b3ff-143f501b64a1.png)

Reply:
Hello @enaburekhan,

Thank you for the detailed review, be sure changes will be implemented.